kani-compiler/src/kani_compiler.rs

```diff
@@ -446,7 +446,14 @@ mod tests {
     use rustc_hir::definitions::DefPathHash;
     use std::collections::HashMap;
 
-    fn mock_next_id() -> HarnessId {
+    fn mock_next_harness() -> HarnessId {
```

Comment:
Still, we're mocking IDs so I think the names should be `mock_next_harness_id` and `mock_next_stub_id`.